packages/rule-performance-budget/src/connections.ts

```diff
@@ -60,7 +58,8 @@ const getConnections = (): Array<NetworkConfig> => {
 
     const configsText = configContent
         .replace(/#.*?\n/g, '') // remove comments
-        .split(`${EOL}${EOL}`);
+        .replace(/\r\n/g, '\n') // remove \r just in case the file has a windows EOL
```

Comment:
Why was this changed?

Reply:
Because the previous code was assuming that a file has the same EOL than the OS, but yesterday that assumption fail :P

Reply:
Ok.